### PR TITLE
Ffm 909

### PR DESCRIPTION
**Pull request by milos85vasic-harness**
This pull request contains the following work:
- Feature config use deprecation
- Consistency for the event and stream URLs.

### CODE REVIEW

File: Sources/ff-ios-client-sdk/API/CfConstants.swift [hunk 1]
```diff
@@ -39,10 +39,12 @@ struct CfConstants
...
 	}
-	struct Server
-	{
-		static let configUrl 	 = "https://config.ff.harness.io/api/1.0"
-		static let eventUrl 	 = "https://config.ff.harness.io/api/1.0/stream" //TODO: Change `config.` -> `event.` once ready
+	
+    struct Server {
+		
+        static let configUrl 	 = "https://config.ff.harness.io/api/1.0"
+		static let streamUrl 	 = "https://config.ff.harness.io/api/1.0/stream"
```
Comment (davejohnston) [inline Sources/ff-ios-client-sdk/API/CfConstants.swift:46]: Some of the formatting in this file and the next seems to be a bit off

**Review by davejohnston — APPROVED** [review]
LGTM.  The indentation in the first files seems a bit funny, but the rest looks good.

File: Sources/ff-ios-client-sdk/API/CfConstants.swift [hunk 2]
```diff
@@ -39,10 +39,12 @@ struct CfConstants
...
 		}
 	}
-	struct Server
-	{
-		static let configUrl 	 = "https://config.ff.harness.io/api/1.0"
-		static let eventUrl 	 = "https://config.ff.harness.io/api/1.0/stream" //TODO: Change `config.` -> `event.` once ready
+	
+    struct Server {
+		
+        static let configUrl 	 = "https://config.ff.harness.io/api/1.0"
```
Comment (enver-bisevac) [inline Sources/ff-ios-client-sdk/API/CfConstants.swift:45]: is this baseUrl?

File: Sources/ff-ios-client-sdk/API/CfConstants.swift (hunk 1 above)
Comment (milos85vasic-harness) [reply Sources/ff-ios-client-sdk/API/CfConstants.swift:46]: @davejohnston Indeed. I will check and update this.

File: Sources/ff-ios-client-sdk/API/CfConstants.swift (hunk 2 above)
Comment (milos85vasic-harness) [reply Sources/ff-ios-client-sdk/API/CfConstants.swift:45]: @enver-bisevac Yes. It is.

**Pull request merged**